op-node/p2p/peer_scores_test.go

```diff
@@ -1,4 +1,4 @@
-package p2p
+package p2p_test
```

Comment:
Need to move the test into `p2p_test` package to avoid cyclic imports caused by the new mock (API). This is the common practice anyway.